packages/elements/src/components/ui/buttons/SecondaryButton.module.css

```
@@ -2,7 +2,7 @@
   /* PrimaryButton overrides */
 
   /* Text */
-  --swui-button-text-color: var(--lhds-color-ui-700);
+  --swui-button-text-color: var(--lhds-color-blue-600);
   --swui-button-text-color-hover: var(--lhds-color-ui-700);
```

Just change this line here! It should be blue-600 and not 500 right?
```suggestion
  --swui-button-text-color-hover: var(--lhds-color-blue-600);
```